### ISSUE TITLE
Saved WiFi passwords not restored

### ISSUE THREAD

**Issue by MyBlackMIDIScore** [seedvault-app/seedvault#288]
I updated my ROM yesterday and I had to wipe my phone in order to do so. After installing the ROM I restored my data from my SeedVault backup on my Nextcloud.
It was my first time putting SeedVault into use, since before that I was using Google's backups for my phone (which also backed up and restored all my saved WiFi passwords) and I thought that SeedVault would do the same (since it also restored all my other settings and configurations). As it turned out though, I checked and none of my passwords were present.

**Comment by grote** [comment]
Wi-Fi passwords get backed up normally. It would help if you can tell us if the settings app was shown as backed up before in backup settings and what version of seedvault you are using.

**Comment by MyBlackMIDIScore** [comment]
Thanks for your reply! 
If I remember correctly, when I restored the backup one of the things it loaded was Device Settings (or Settings Configuration, I don’t remember exactly because I didn’t pay much attention to it then). 
I’m also not sure about the version of Seedvault used to backup the data (if there is any way to find out through .backup.metadata let me know because I tried opening it with a text editor and it was all gibberish), but the version that I restored the backup with is 11-1.1.
I also still have the full backup on my Nextcloud, so if there is anything I can provide that may help, let me know.
